bootstrap.go

```diff
@@ -9,16 +9,22 @@ import (
 	"gopkg.in/DataDog/dd-trace-go.v1/profiler"
 )
 
+type ConnectionType int
```

Comment:
If this is config, then it must be in `config` pkg

Reply:
As it is currently used it is not config, it is just a parameter for functions.

Reply:
@nhhagen  why `int` why not `byte`? It's enum and you will not exide a connection types more that 255, so change it to smaller size

Reply:
I can make it `byte`